frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +244,25 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbs = (model, kindObj, match) => {
+  const lastNamespace = getLastNamespace();
+  return [
+    {
+      name: i18next.t(model.labelPluralKey) || model.labelPlural,
+      path: resourceListPathFromModel(
+        model,
+        !lastNamespace || lastNamespace === ALL_NAMESPACES_KEY ? null : lastNamespace,
+      ),
+    },
+    {
+      name: i18next.t('details-page~{{kind}} details', {
+        kind: i18next.t(kindObj.labelKey) || kindObj.label,
```

Comment:
```suggestion
        kind: kindObj.labelKey ? i18next.t(kindObj.labelKey) : kindObj.label,
```

You might want to assign this to a variable above to avoid having to check this twice.

Reply:
> You might want to assign this to a variable above to avoid having to check this twice.

@spadgett it's not the same variable.  One is getting label from model, the other from kindObj.   Ie. One is `model.labelPluralKey` and the other is `kindObj.labelKey`